apps/beeswax/src/beeswax/server/hive_server2_lib_tests.py

```diff
@@ -298,7 +298,7 @@ def test_get_table_with_error(self):
         client.get_table(database='database', table_name='table_name')
       except QueryServerException as e:
         if sys.version_info[0] > 2:
-          req_string = "TGetTablesReq(sessionHandle=TSessionHandle(sessionId=THandleIdentifier(guid=b'1', secret=b'1')), catalogName=None, schemaName='database', tableName='table_name', tableTypes=None)"
+          req_string = "TGetTablesReq(sessionHandle=TSessionHandle(sessionId=THandleIdentifier(guid=b'l\\xc4', secret=b'l\\xc4')), catalogName=None, schemaName='database', tableName='table_name', tableTypes=None)"
         else:
```

Comment:
for this also @romainr

Reply:
If you type Hive or Impala queries in a live Hue, does it work?

It feels like we double encode

Reply:
yes, with this branch locally, Hive queries are working.
<img width="1790" alt="Screenshot 2021-01-12 at 9 24 03 PM" src="https://user-images.githubusercontent.com/36241930/104338427-809ab580-551c-11eb-8984-e32726114afd.png">

Reply:
tracking @ https://github.com/cloudera/hue/pull/1621